tests/e2e/server/env.config.copy.mjs

```diff
@@ -18,7 +18,8 @@ const oauth = {
   scope: 'openid profile me.read',
 };
 const origins = {
-  app: 'https://sdkapp.example.com',
+  // Ensure all domains are added to the security cert creation
+  app: process.env.LIVE ? 'https://sdkapp.petrov.ca' : 'https://sdkapp.example.com',
```

Comment:
This will allow us to do live automation on our localhost with Webkit's more restrictive "Website Tracking" preference.